devices.js

```diff
@@ -4501,6 +4501,57 @@ const devices = [
         },
         exposes: [e.temperature(), e.humidity(), e.illuminance(), e.illuminance_lux(), e.soil_moisture(), e.pressure(), e.battery()],
     },
+    {
+        zigbeeModel: ['DIYRuZ_AirSense'],
+	    model: 'DIYRuZ_AirSense',
+        vendor: 'DIYRuZ',
+        description: '[Air quality sensor](https://modkam.ru/?p=1715)',
```

Comment:
Please add `exposes` here (look at other devices for examples) and fix the lint errors (`npm run lint`).

Reply:
Done!